app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt

```diff
@@ -586,9 +557,12 @@ object ImageUtil {
             return false
         }
 
-        Timber.d(
-            "Splitting image with height of $imageHeight into $partCount part with estimated ${optimalSplitHeight}px height per split",
-        )
```

Comment:
why is this removed too?

Reply:
I was intending to move it down to the new method for splitting.